bricks_inline/bricks_inline.module

```diff
@@ -0,0 +1,247 @@
+        '#size' => 1,
+        '#attributes' => array(
+          'class' => array('ief-entity-depth'),
+        ),
+      );
+        $entities[$delta]['options'] = array(
+        '#tree' => 'true',
+        '#weight' => 11,
+        'classes' => array(
+          '#type' => 'textfield',
+          '#attributes' => array(
+            'placeholder' => t('Additional CSS classes'),
+          ),
+          '#default_value' => !empty($item['options']['classes']) ? $item['options']['classes'] : '',
+        ),
+      );
+    }
+  }
+}
+
+/**
+ * Implements hook_field_attach_submit().
+ */
+function bricks_inline_field_attach_submit($parent_entity_type, $parent_entity, $form, &$form_state) {
+  list(, , $bundle_name) = entity_extract_ids($parent_entity_type, $parent_entity);
+
+  foreach (field_info_instances($parent_entity_type, $bundle_name) as $instance_name => $instance) {
+    if (isset($instance['widget']) && strpos($instance['widget']['type'], 'inline_entity_form') === 0) {
+      $field_name = $instance['field_name'];
+      if (!isset($form[$field_name])) {
+        // The field wasn't found on this form, skip it.
+        // Usually happens on stub entity forms that don't contain all fields.
+        continue;
+      }
+      $langcode = $form[$field_name]['#language'];
+      $ief_id = $form[$field_name][$langcode]['#ief_id'];
+      if (empty($form_state['inline_entity_form'][$ief_id])) {
+        // No data found, no need to do anything.
+        continue;
+      }
+      $values = $form_state['inline_entity_form'][$ief_id];
+      $entity_type = $values['settings']['entity_type'];
+
+      $entities = $form_state['values'][$field_name][$langcode]['entities'];
+      uasort($values['entities'], 'drupal_sort_weight');
+      uasort($entities, 'drupal_sort_weight');
+
+      foreach ($values['entities'] as $delta => $item) {
+        list($entity_id) = entity_extract_ids($entity_type, $item['entity']);
+        $entity_ids[] = array(
+          $values['settings']['column'] => $entity_id,
+          'depth' => $entities[$delta]['depth'],
+          'options' => $entities[$delta]['options'],
+        );
+      }
+
+      if (!empty($entity_ids)) {
+        // Set the list of ids as the field value.
+        $parent_entity->{$field_name}[$langcode] = $entity_ids;
+      }
+    }
+  }
+}
+
+/**
+ * Implements hook_module_implements_alter().
+ */
+function bricks_inline_module_implements_alter(&$implementations, $hook) {
+  if ($hook == 'field_attach_submit') {
+    // Move bricks_inline_field_attach_submit() at the end of the list of implementations.
+    $group = $implementations['bricks_inline'];
+    unset($implementations['bricks_inline']);
+    $implementations['bricks_inline'] = $group;
+  }
+}
+
+/**
+ * Theme function for inline_entity_form field widget.
+ *
+ * @see bricks_inline_field_widget_form_alter()
+ */
+function theme_bricks_inline_entity_form_table($variables) {
```

Comment:
As I figured out- here we have a copy-paste from http://cgit.drupalcode.org/inline_entity_form/tree/inline_entity_form.module?h=7.x-1.8#n1659 + modifications similar to https://github.com/berdyshev/drupal-bricks/blob/7.x-5.x/bricks.edit.inc#L44.

Let's split this function in two parts:
1. `_template_preprocess_inline_entity_form_entity_table` - copy-paste from IEF.
2. `_bricks_preprocess_tabledrag_form ` helper like in 8.x https://github.com/highweb/drupal-bricks/blob/8.x-1.x/bricks.module#L179.
3. And then call 1) and 2) in `bricks_inline_preprocess_inline_entity_form_entity_table` like in 8.x: https://github.com/highweb/drupal-bricks/blob/8.x-1.x/modules/bricks_inline/bricks_inline.module#L25.

So, we can submit a patch in IEF and remove `_template_preprocess_inline_entity_form_entity_table` completely once accepted.